src/app/shared/stache/modules/code-block/code-block.component.ts

```diff
@@ -4,6 +4,7 @@ declare let Prism: any;
 import 'prismjs/prism';
 import 'prismjs/components/prism-typescript';
 import 'prismjs/plugins/normalize-whitespace/prism-normalize-whitespace';
+require('style-loader!prismjs/themes/prism.css');
```

Comment:
Skyux cannot import the module anywhere.  The modules in this case are specifically for our exported StacheModule.    The components that get pulled into the skyAppModule still need access to this, and it blows up if we require the module.